### PR TITLE
Handle unknown commands

### PR DESCRIPTION
**Pull request by kziemianek**
Just to not fail silently. If you have more appropriate message please share :)

### CODE REVIEW

File: packages/cli/src/hooks/command_not_found/command_moved.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ const hook: Hook<"command_not_found"> = async function (opts) {
     process.stdout.write(
       `You can use it like: ${chalk.greenBright(`swanky contract ${opts.id} contract_name`)}\n`
```
Comment (kziemianek) [inline packages/cli/src/hooks/command_not_found/command_moved.ts:10]: I think this file should be renamed as it handles more scenarios than before.

Comment (shunsukew) [reply packages/cli/src/hooks/command_not_found/command_moved.ts:10]: I think so too.

`packages/cli/src/hooks/command_not_found.ts` and modify package.json like below?

```
"hooks": {
      "command_not_found": [
        "./dist/hooks/command_not_found"
      ]
    }
```

**Review by shunsukew — CHANGES_REQUESTED** [review]
I think `package.json` should be also updated if file name has been changed.
https://github.com/AstarNetwork/swanky-cli/blob/09cfa063b3c35c582191ac6595d8dda0d7c28025/packages/cli/package.json#L86

**Comment by kziemianek** [comment]
> I think `package.json` should be also updated if file name has been changed.
> 
> https://github.com/AstarNetwork/swanky-cli/blob/09cfa063b3c35c582191ac6595d8dda0d7c28025/packages/cli/package.json#L86

You've already pointed this out, don't know how I could miss it.

**Review by shunsukew — APPROVED** [review]
Thank you

**Pull request merged**